src/test/datascience/nativeEditor.functional.test.tsx

```diff
@@ -94,7 +94,7 @@ suite('Dummy8', () => {
 // function waitForMessage(ioc: DataScienceIocContainer, message: string, options?: WaitForMessageOptions): Promise<void> {
 //     return ioc.getNativeWebPanel(undefined).waitForMessage(message, options);
 // }
-// suite('DataScience Native Editor', () => {
```

Comment:
is the 'functional' test suite something we can just drop?

Reply:
Yes, will do in another PR